en/advanced_config/racer_setup.md

```diff
@@ -0,0 +1,200 @@
+# Racer Setup
```

Comment:
@bkueng This is awesome! I've changed some of the heading levels. You might want to check they make sense still.